pkg/config/config_template.yaml

```diff
@@ -3163,6 +3163,7 @@ api_key:
     ## @param enabled - boolean - optional - default: true
     ## @env DD_OTLP_CONFIG_METRICS_ENABLED - boolean - optional - default: true
     ## Whether to ingest metrics through the OTLP endpoint. Set to false to disable OTLP metrics ingest.
+    ## Note that you need to set the otlp_config.receiver section in order to enable OTLP in the Agent.
```

Comment:
```suggestion
    ## Set to false to disable metrics support in the OTLP ingest endpoint.
    ## This does not enable/disable OTLP ingest for other datatypes, use the otlp_config.receiver section for this.
```

Reply:
It's not that this doesn't enable it for "other datatypes", it's that this doesn't enable it at all.

Reply:
```suggestion
    ## Set to false to disable metrics support in the OTLP ingest endpoint.
    ## To enable the OTLP ingest, the otlp_config.receiver section must be set.
```
Maybe?

Reply:
That works for me :)